### DISCUSSION TITLE
Name: Tanomu yo?

### DISCUSSION THREAD

**Discussion by earthboundkid** [earthboundkid/requests#48] [Ideas]
I never could come up with a good pun name for requests, so I just used requests, the same as Python, but in retrospect, it should have been called tanomu http://neojaponisme.com/2008/09/23/tanomu-yo/ https://archive.org/details/Xin10/. Maybe use the name if there’s ever a 2.0 API.

**Comment by earthboundkid** [comment]
Another funny name would be something along the lines of Carl → curl. Maybe kurl or carlurl?

**Reply by earthboundkid** [reply]
Curlanaurl

**Comment by raeperd** [comment]
There are several package names I had in mind when forking this repo. I'll leave it here in case it can be of help.
- xquery
- httpx 
- gurl (for c-url to go-url)

x was named after context which is my favorite feature in this package

**Reply by earthboundkid** [reply]
"Gurl" is funny.
